python/mujinplanningclient/packingplanningclient.py

```diff
@@ -56,17 +60,30 @@ def StartPackFormationComputationThread(self, timeout=10, debuglevel=4, toolname
         taskparameters.update(kwargs)
         return self.ExecuteCommand(taskparameters, toolname=toolname, timeout=timeout)
 
-    def StartSingleSkuPackFormationComputationThread(self, packingParameters, packLocationName, toolname=None, timeout=10, **kwargs):
+    def StartSingleSKUPackFormation(
+        self,
+        userPackFormationComputationParameters,
+        systemState,
+        toolname=None,
```

Comment:
I think we should agree on a consistent capitalization of `toolname`, at least for the clients. In validation API it's `toolName`, even though it's immediately converted to `toolname` when passed down to a packing task.

Reply:
Sure.